troposphere/static/js/components/images/detail/versions/Version.react.js

```diff
@@ -21,29 +26,42 @@ export default React.createClass({
         showAvailability: React.PropTypes.bool
     },
 
-    getDefaultProps: function() {
+    getDefaultProps() {
         return {
             showAvailability: true,
             editable: true
         }
     },
 
-    onEditClicked: function() {
+    onCardClick() {
+        let isOpen = this.state.isOpen
+            ? false : true;
```

Comment:
Welp, i just read to fast. But what caught my eye, was the use of looking at a boolean, and returning a boolean literal, which can always be simplified.